content-services/latest/upgrade/index.md

```diff
@@ -47,7 +47,7 @@ The following table shows the upgrade path for major versions:
 | Community Edition 7.2 | Content Services 7.2 |
 | Community Edition 7.3 | Content Services 7.3 |
 | Community Edition 7.4 | Content Services 7.4 |
-| Community Edition 23.1 | Content Services 23.1 |
+| Community Edition 23.1 | Content Services 23.x |
```

Comment:
Let's list the two versions: 

- Community Edition 23.1 .. Content Services 23.1
- Community Edition 23.2 .. Content Services 23.2